### PR TITLE
Feature/things to know

### PR DESCRIPTION
**Pull request by borulebaja**

### CODE REVIEW

**Review by mrterranova — APPROVED** [review]
Thanks for the locations table edit

File: EarthBnB/src/main/resources/application.properties [hunk 1]
```diff
@@ -7,10 +7,10 @@ spring.flyway.baseline-on-migrate = true
 spring.flyway.check-location=true
 spring.flyway.url = jdbc:mysql://localhost:3306/earthdb?autoreconnect=true
 spring.flyway.username = root
-spring.flyway.password = root
+spring.flyway.password = password
```
Comment (avalojc) [inline EarthBnB/src/main/resources/application.properties:10]: did you mean to commit this? Wont this cause issues with everyone elses?

File: EarthBnB/src/main/resources/application.properties [hunk 2]
```diff
@@ -7,10 +7,10 @@ spring.flyway.baseline-on-migrate = true
...
 spring.flyway.enabled=true
 
 spring.datasource.url = jdbc:mysql://localhost:3306/earthdb?autoreconnect=true
 spring.datasource.username = root
-spring.datasource.password = root
+spring.datasource.password = password
```
Comment (avalojc) [inline EarthBnB/src/main/resources/application.properties:15]: same as above

**Review by avalojc — CHANGES_REQUESTED** [review]
2 things, 
-why is gitignore deleted?
-by committing your change in password you are changing everyones password

**Pull request closed without merging**